packages/airnode-node/src/constants.ts

```diff
@@ -22,22 +22,22 @@ export const BLOCK_MIN_CONFIRMATIONS = 0;
 export const CONVENIENCE_BATCH_SIZE = 10;
 
 // The default amount of time before a "retryable" promise is timed out and retried
-export const DEFAULT_RETRY_TIMEOUT_MS = 5_000;
+export const DEFAULT_RETRY_TIMEOUT_MS = 30_000;
```

Comment:
Why change this one? I think this is used in quite a lot of places throughout the app where a promise is retried and could change behaviour quite significantly. I would rather call the places we want to set something other than the default of 5 seconds

Reply:
I agree, but I'd actually refactor this constant to be more contextual. From what I see we use it only to limit blockchain provider calls. Maybe renaming it to `BLOCKCHAIN_PROVIDER_TOTAL_TIMEOUT` ?

Actually I think we should not use go utils for this at all and use https://github.com/lifeomic/attempt#readme instead (which we already use).

I've created https://api3dao.atlassian.net/browse/AN-526

Reply:
I'll revert this change.